elements/rh-stat/rh-stat.css

```diff
@@ -1,5 +1,5 @@
 :host([on='dark']) {
-    --accent-color: var(--rh-color-brand-red-on-dark, #ff3333);
+    --accent-color: var(--rh-color-brand-red-on-dark, #fff);
```

Comment:
Instead of changing the default value for the `--rh-color-brand-red-on-dark` token, which is defined in the tokens repo, you should set the value directly on `--accent-color`. I also suggest renaming to either `--rh-stat-accent-color` (if intended to be public) or `--_accent-color` (if intended to be private). In the latter case, I also suggest assigning the property on the shadow container, instead of the host.

This should maybe wait until after https://github.com/RedHat-UX/red-hat-design-system/pull/471 then rebase on top of that

Reply:
Updated this based on the feedback & rebased.